docs/langgraph/multi-agent.md

```diff
@@ -1,16 +1,376 @@
 # Multi-agent Systems
 
-## Agent Communication Patterns
-<!-- Agent-to-agent messaging -->
-<!-- Shared state management -->
-<!-- Coordination mechanisms -->
-
-## Supervisory Patterns
-<!-- Supervisor agent setup -->
-<!-- Task delegation -->
-<!-- Result aggregation -->
-
-## Multi-turn Conversations
-<!-- Cross-agent conversations -->
-<!-- Context preservation -->
-<!-- Turn management -->
\ No newline at end of file
+In this how-to guide we will demonstrate how to implement a multi-agent network architecture where each agent can communicate with every other agent (many-to-many connections) and can decide which agent to call next. We will be using LangGraph's functional API — individual agents will be defined as tasks and the agent handoffs will be defined in the main entrypoint.
+
+## Agent Communication Patterns & Supervisory Patterns & Multi-turn Conversations
+
+### How to build a multi-agent network (functional API)
+
+In this how-to guide we will demonstrate how to implement a multi-agent network architecture where each agent can communicate with every other agent (many-to-many connections) and can decide which agent to call next.
+
+```typescript
+import { entrypoint, task } from "@langchain/langgraph";
+import { createReactAgent } from "@langchain/langgraph/prebuilt";
```

Comment:
ref:
https://langchain-ai.github.io/langgraphjs/how-tos/multi-agent-network-functional/

I'm hesitant because we don't want to use `createReactAgent`.💭

Reply:
I updated `docs/langgraph/README.md` and added instructions to avoid using `createReactAgent`.
ceb1da01df8dbf4dc432a6941d888c107d07c3c2

Reply:
Although it is documented, but we do not use it, so it would be better not to add this section.
We also do not use the Functional API.